bundles/org.openhab.binding.fronius/src/main/java/org/openhab/binding/fronius/internal/api/FroniusBatteryControl.java

```diff
@@ -208,4 +214,36 @@ ScheduleType.CHARGE_MIN, new TimeTableRecord(from.format(TIME_FORMATTER), until.
         timeOfUse[timeOfUse.length - 1] = holdCharge;
         setTimeOfUse(new TimeOfUseRecords(timeOfUse));
     }
+
+    /**
+     * Sets the reserved battery capacity for backup power.
+     *
+     * @param percent the reserved battery capacity for backup power
+     * @throws FroniusCommunicationException when an error occurs during communication with the inverter
+     * @throws IllegalArgumentException if the percent is not in [10,95]
+     * @throws FroniusUnauthorizedException when the login failed due to invalid credentials
+     */
+    public void setBackupReservedCapacity(PercentType percent)
```

Comment:
Just a quick comment: It seems strange to take an openHAB `PercentType` here in the API class where only the `intValue` is used. I would propose:
```suggestion
    public void setBackupReservedCapacity(int percent)
```